### ISSUE TITLE
[Bug] Accessibility problem with hidden views

### ISSUE THREAD

**Issue by Harwinder0501** [xamarin/Xamarin.Forms#6929]
### Description
After Updating project Xamarin Forms 4.1 Accessiblity for the view those are hidden initially and visible later are not accessible.   
### Steps to Reproduce

1.  Add two labels to a page. keep one label hidden.
2. Add one button to control visibility of hidden label.
3. Enable Voice over feature in IOS and load the page.
4. Click on button so that it hidden label show on page. 
Check the behaviour of accessibility, it won't announce second label.

### Expected Behavior
Voice over should work for hidden view when they are visible.

### Actual Behavior
Voice over not working for hidden view when they are visible.

### Basic Information

- Version with issue: Xamarin 4.0 onwards.
- Last known good version: 
- IDE: Visual Studio For MAC
- Platform Target Frameworks: 
  - iOS:  8.0

**Comment by jsuarezruiz** [comment]
I attach the sample.
[Test6929.zip](https://github.com/xamarin/Xamarin.Forms/files/3411122/Test6929.zip)

**Comment by rmarinho** [comment]
This works fine for me on iOS . Steps i used

Read label, slide next , Reads button, tab button, slide back reads label, slide back reads invisible label.

Please change the sample to show your issue. Thanks

**Comment by pamela032709** [comment]
Accesibility tags are not being read within the WebViews. something happened with Xamarin forms new version that doesn't let voice over identify the content.I reported and no answer.
To test add this line to any xaml
    <WebView Source="https://twitter.com" AutomationProperties.IsInAccessibleTree="true" AutomationProperties.HelpText="Tap to toggle the activity indicator" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"></WebView>

**Comment by samhouts** [comment]
@Harwinder0501 Can you please check the sample attached to https://github.com/xamarin/Xamarin.Forms/issues/6929#issuecomment-513216437 and alter it such that it demonstrates the issue? Thanks!

@pamela032709 We see your issue logged as #6894. It's possibly the same root cause. Thanks!

**Comment by pamela032709** [comment]
Thank you for the info. I tried that but the webviews still not been read by voice over. when do you think this fix will be ready?

**Comment by Harwinder0501** [comment]
@rmarinho  @samhouts I checked the attached sample. Although the example is working fine, but with further investigation I am able to reproduce the issue.  

To reproduce it:
in MainPage.Xaml, Remove Vertical option from the stack layout.

**Comment by ryl** [comment]
Running into this problem as well.

**Comment by SravanthiKare** [comment]
@samhouts @rmarinho 
Can you please let us know the release date for the fix? We have a release which is dependent on this fix.

**Comment by samhouts** [comment]
I do not have an estimated release date yet, but it is under active investigation. Thank you for your patience!

**Comment by Harwinder0501** [comment]
Hey @samhouts  , Can you please confirm under which release we will get this fix ?

**Comment by SravanthiKare** [comment]
Hi @samhouts , 
Can you please share the estimated release date for this issue, as we are highly dependent on it?

**Comment by pamela032709** [comment]
same on this side. Im totally depending on this fix at this point.

**Comment by samhouts** [comment]
It is available today! 

https://www.nuget.org/packages/Xamarin.Forms/4.1.0.778454 

Thank you for your patience.

**Comment by Harwinder0501** [comment]
@samhouts  In which version we can expect this fix.

**Comment by jfversluis** [comment]
@Harwinder0501 It should be available at least in 4.1.0.778454, 4.2.0.778463 and 4.3.0.778476-pre1. Is it not working for you?

**Comment by pamela032709** [comment]
I tested and it is working with the latest xamarin forms version.

**Comment by boyd-m** [comment]
We are still observing problems with this on iOS in 4.2.0.778463. Although the item can be selected by tapping on it directly, it can not be selected by swiping left or right.

**Comment by jfversluis** [comment]
@boyd-m since some people have reported their problems are gone, could you please create a new issue (make sure to reference this one if you think it's releated) so we can track it better? 

Thanks!

**Comment by boyd-m** [comment]
I've created a sample project which demonstrates the issue: https://github.com/boyd-m/VisibilityA11y
It shows the same problem described in the original description of this bug, so should this remain on here? Happy to raise the bug again if not. The sample project was using 4.2.0.815419

**Comment by jfversluis** [comment]
@boyd-m seeing this issue is a bit littered with some other things I would recommend opening a new one if it's not too much trouble. Big kudos on already creating a reproduction, thank you for that!

**Comment by boyd-m** [comment]
@jfversluis No problem, I've raised https://github.com/xamarin/Xamarin.Forms/issues/7606 to cover this

**Comment by pamela032709** [comment]
@samhouts Now , I experience an issue with the MasterDetailPage when the menu is hidden . voice over reads everything on the page plus whatever was on the menu items. Should I create a new bug  or there is something I could do to make the master detail page not active in voice over when the menu is not present?

**Comment by samhouts** [comment]
@pamela032709 I think you're experiencing https://github.com/xamarin/Xamarin.Forms/issues/4220
